pheweb/serve/templates/gene.html

```diff
@@ -40,28 +56,37 @@ <h1 style="margin-top:0">{{ gene_symbol }}</h1>
 <div class="row">
   <div class="col-md-10 col-lg-10 col-sm-10 col-xs-10">
     <p>Phenotypes with the most-significant associations for this locus:</p>
-
+  </div>
+</div>
+<div class="row">
+  <div class="col-xs-12 col-sm-8 col-md-6">
+    <input type="text" class="form-control" id="search" placeholder="Search... &quot;coronary&quot;, &quot;nervous system&quot;, etc.">
+  </div>
+  <div class="col-xs-12 col-sm-4 col-sm-6">
+    <h5 class="pull-right"><div id="streamtable-found" class="label label-primary"></div></h5>
+  </div>
+</div>
       <div class="row stream_table_row">
         <div class="col-xs-12">
           <table id="stream_table" class="table table-striped table-bordered">
             <thead>
               <tr>
-                <th>Top p-value in gene</th>
-                <th>Phenotype</th>
+                <th style="width: 7%;">p-value</th>
+                <th style="width: 15%;">variant</th>
+                <th>phenotype</th>
+                <th>category</th>
+                <th style="width: 7%;">number of cases</th>
               </tr>
             </thead>
             <tbody>
             </tbody>
           </table>
-          <div class='btn'>
+          <div class='btn' style='padding-left: 0px'>
```

Comment:
Add configuration what fields are exported. Now exports everything including paths to pheweb files.   Add the configuration to conf_utils.py. Look at 'var_top_pheno_export_fields' in variant.js for example...